Buildings/Applications/DHC/EnergyTransferStations/CoolingIndirect.mo

```diff
@@ -0,0 +1,316 @@
+within Buildings.Applications.DHC.EnergyTransferStations;
+model CoolingIndirect
+  "Indirect cooling energy transfer station for district energy systems"
+  extends Buildings.Fluid.Interfaces.PartialFourPort(
+    redeclare final package Medium1 = Medium,
+    redeclare final package Medium2 = Medium);
+
+  replaceable package Medium =
+    Modelica.Media.Interfaces.PartialMedium "Medium in the component";
+
+  // mass flow rates
+  parameter Modelica.SIunits.MassFlowRate m1_flow_nominal(min=0,start=0.5)
+    "Nominal mass flow rate of primary (district) district cooling side";
+  parameter Modelica.SIunits.MassFlowRate m2_flow_nominal(min=0,start=0.5)
+    "Nominal mass flow rate of secondary (building) district cooling side";
+
+  // Heat exchanger
+  parameter Modelica.SIunits.PressureDifference dp1_nominal(min=0,start=500)
+    "Nominal pressure difference on primary side"
+    annotation(Dialog(enable=true,
+      tab="Heat exchanger"));
+  parameter Modelica.SIunits.PressureDifference dp2_nominal(min=0,start=500)
+    "Nominal pressure difference on secondary side"
+    annotation(Dialog(enable=true,
+      tab="Heat exchanger"));
+  parameter Boolean use_Q_flow_nominal=true
+    "Set to true to specify Q_flow_nominal and temperatures, or to false to specify effectiveness"
+    annotation(Dialog(enable=true,
+      tab="Heat exchanger"));
+  parameter Modelica.SIunits.HeatFlowRate Q_flow_nominal(min=0,start=10000)
+    "Nominal heat transfer"
+    annotation(enable=true,
+      Dialog(tab="Heat exchanger"));
+  parameter Modelica.SIunits.Temperature T_a1_nominal(min=0+273,max=100+273.15,start=5+273.15)
+    "Nominal temperature at port a1"
+    annotation(Dialog(enable=true,
+      tab="Heat exchanger"));
+  parameter Modelica.SIunits.Temperature T_a2_nominal(min=0+273,max=100+273.15,start=7+273.15)
+    "Nominal temperature at port a2"
+    annotation(Dialog(enable=true,
+      tab="Heat exchanger"));
+  parameter Modelica.SIunits.Efficiency eta(min=0,max=1)=0.8
+    "Constant effectiveness"
+    annotation(Dialog(enable=true,
+      tab="Heat exchanger"));
+
+  // Primary supply control valve
+  parameter Modelica.SIunits.PressureDifference dpValve_nominal(min=0)=50
+    "Control valve: Nominal pressure drop of fully open valve";
+
+  // Secondary supply pump
+  parameter Modelica.SIunits.PressureDifference dp_nominal(min=0)
```

Comment:
This parameter is not used below. May not need to provide here.

Reply:
You're right. This can be removed.